src/app/(protected)/super-admin/organizations/[id]/page.tsx

```
@@ -0,0 +1,329 @@
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select';
+import { useToast } from '@/hooks/use-toast';
+import { ArrowLeft, Shield, User, Stethoscope } from 'lucide-react';
+import { format } from 'date-fns';
+
+interface User {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)